src/models/index.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const { Sequelize, DataTypes } = require('sequelize');
+const clothesModel = require('./clothes/model.js');
+const foodModel = require('./food/model.js');
+const Collection = require('./data-collection.js');
+
+const DATABASE_URL = process.env.DATABASE_URL || 'sqlite:memory:';
+
+const sequelize = new Sequelize(DATABASE_URL);
```

Comment:
You are trying to use two different database instances, creating one here and one in the auth models file. This should be a singleton; create one instance and have anything else refer to that one instance.